management/server/http/api/openapi.yml

```
@@ -1806,6 +1821,38 @@ paths:
           "$ref": "#/components/responses/forbidden"
         '500':
           "$ref": "#/components/responses/internal_error"
+  /api/peers/{peerId}/accessible-peers:
+    get:
+      summary: List of accessible peers
+      description: Returns a list of peers that the specified peer can connect to within the network.
+      tags: [ Accessible Peers ]
```

```suggestion
      tags: [ Peers ]
```
As we decided to keep the enpoint together with the peers tab I would like to have the same for the docs. This tag will decide wether we add it to the Peers dropdown in doc or create a new page.